src/main/java/mate/model/Car.java

```diff
@@ -1,13 +1,14 @@
 package mate.model;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 
 public class Car {
     private Long id;
     private String model;
     private Manufacturer manufacturer;
-    private List<Driver> drivers;
+    private List<Driver> drivers = new ArrayList<>();
```

Comment:
An entity can be with not initialized files. There can be a confusing situation, or we get a car without drivers or the drivers field is just not initialized. Let's leave `null` in case it was not initialized

Reply:
Also, there is a rule that initialization logic must be inside the constructor or setter method